Author: Pino Toscano <pino@debian.org>
Description: Respect the LIB_DESTINATION cmake variable
 Use the LIB_DESTINATION in install() calls, instead of hardcoding lib$SUFFIX
 everywhere.
Last-Update: 2012-01-16
Forwarded: no

---
 CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libindi/CMakeLists.txt
===================================================================
--- libindi.orig/CMakeLists.txt	2013-11-06 17:30:59.715963317 +0100
+++ libindi/CMakeLists.txt	2013-11-06 17:30:59.711963476 +0100
@@ -25,7 +25,7 @@
 set (LIB_DESTINATION "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}" CACHE STRING "Library directory name")
 ## the following are directories where stuff will be installed to
 set(INCLUDE_INSTALL_DIR      "${CMAKE_INSTALL_PREFIX}/include/" CACHE PATH "The subdirectory to the header prefix")
-set(PKGCONFIG_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX}/pkgconfig/" CACHE STRING "Base directory for pkgconfig files")
+set(PKGCONFIG_INSTALL_PREFIX "${LIB_DESTINATION}/pkgconfig/" CACHE STRING "Base directory for pkgconfig files")
 
 ##################  Includes  ################################
 Include (CheckCXXSourceCompiles)
